release-notes/opensearch-sql.release-notes-2.0.0.0-rc1.md

```diff
@@ -0,0 +1,37 @@
+### Version 2.0.0.0-rc1 Release Notes
+Compatible with OpenSearch and OpenSearch Dashboards Version 2.0.0.0-rc1
+
+### Features
+* Add Kmeans and AD command documentation ([#493](https://github.com/opensearch-project/sql/pull/493))
+* AD and Kmeans grammar edits ([#500](https://github.com/opensearch-project/sql/pull/500))
+* Support more parameters for AD and KMEANS command, and update related documentation ([#515](https://github.com/opensearch-project/sql/pull/515))
+
+### Maintenance
+* Version 2.0 ([#507](https://github.com/opensearch-project/sql/pull/507))
+* Removed changes introduced to support JDK8 ([#513](https://github.com/opensearch-project/sql/pull/513))
+* Add JDK 17 support ([#512](https://github.com/opensearch-project/sql/pull/512))
+* Fix bwc build issue with jdk17 ([#520](https://github.com/opensearch-project/sql/pull/520))
```

Comment:
minor: tests related can go to infrastructure category, maintenance would be plugin version changes to maintain upstream compatibility